emscripten.py

```diff
@@ -621,7 +621,7 @@ def optimize_syscalls(declares, DEBUG):
     syscall_prefix = '__syscall'
     syscall_numbers = [d[len(syscall_prefix):] for d in declares if d.startswith(syscall_prefix)]
     syscalls = [int(s) for s in syscall_numbers if is_int(s)]
-    if set(syscalls).issubset(set([6, 54, 140, 146])): # close, ioctl, llseek, writev
```

Comment:
This seems like it changes the behaviour.    Should we try to preserve this?

Reply:
I think this does not change the behavior. The property tested there is that the musl syscalls fall in that set - "no fancy syscalls are used". We no longer have 146 at all, so it's ok to remove it from there, and we still check that "fancy syscalls" property as before.